src/routes.json

```diff
@@ -424,10 +424,10 @@
         "title": "micro:bit"
     },
     {
-        "name": "banned-splash",
-        "pattern": "^/accounts/banned-response/?(\\?.*)?$",
-        "routeAlias": "/accounts/banned-response/?$",
-        "view": "banned-splash/banned-splash",
+        "name": "bad-username-splash",
+        "pattern": "^/accounts/bad-username/?(\\?.*)?$",
+        "routeAlias": "/accounts/bad-username/?$",
```

Comment:
Could this be just /bad-username/ ? The /accounts/ namespace is mainly used by scratchr2 so putting it in here could be confusing